readme.md

```diff
@@ -1,13 +1,95 @@
 # Weaviate website
 
-This repo contains the [Weaviate](https://weaviate.io) website, it's built using Jekyll.
+This repo contains the [Weaviate](https://weaviate.io) website, it's built using [Jekyll](https://jekyllrb.com/).
 
-### Run locally
+## Installing & Running
 
-```
-$ bundle install
-$ bundle exec jekyll serve
-```
+### Dependencies
+
+- Software Dependencies
+  - [Ruby](https://www.ruby-lang.org/en/documentation/installation/)
+
+- Installing Dependencies in **Ubuntu/Debian**
+  ```bash
+  # update repositories & install basic build dependencies
+  sudo apt update && sudo apt install -y build-essential bash git rsync
+
+  # update repositories & install rbenv
+  sudo apt update && sudo apt install rbenv ruby-build
+
+  # initialize rbenv
+  rbenv init
+
+  # updates ruby-build local packages
+  git clone https://github.com/rbenv/ruby-build.git "$(rbenv root)"/plugins/ruby-build
+
+  # install ruby 2.7.5
+  rbenv install 2.7.5
+
+  # set the ruby version locally to 2.7.5
+  rbenv local 2.7.5
+  ```
+  - For installing RubyGems, read [docs](https://jekyllrb.com/docs/installation/ubuntu/)
+  ```bash
+  echo '# Install Ruby Gems to ~/gems' >> ~/.bashrc
+  echo 'export GEM_HOME="$HOME/gems"' >> ~/.bashrc
+  echo 'export PATH="$HOME/gems/bin:$PATH"' >> ~/.bashrc
+  source ~/.bashrc
+  ```
+
+- Installing Dependencies Windows
+  - Please use [WSL](https://docs.microsoft.com/en-us/windows/wsl/)
+
+- Installing Dependencies in macOS
+  ```bash
+  # install dependencies
+  brew install rsync openssl rbenv ruby-build
+
+  # install ruby 2.7.5
+  rbenv install 2.7.5
+  
+  # set the ruby version locally to 2.7.5
+  rbenv local 2.7.5
+  ```
+
+- Check if dependencies are installed correctly
+  ```bash
+  ruby -v
+  gem -v
+  ```
+
+- Finally, install Jekyll and Bundler:
+  ```bash
+  gem install jekyll bundler
+  ```
+
+- Check if dependencies are installed correctly
+  ```bash
+  jekyll -v
+  ```
+### Setting up the repository
+  
+- To get the site up and running locally, follow the below steps:
+
+  **PS.:** You need to have a full Bash environment. If you're on Windows, please use WSL.
+
```

Comment:
Could you add the link to Microsoft Docs which tells how to activate WSL on windows here.
Link - [WSL](https://docs.microsoft.com/en-us/windows/wsl/install)